### PR TITLE
Root contract

### PR DESCRIPTION
**Pull request by hujw77**

### CODE REVIEW

**Review by hackfisher — APPROVED** [review]
@hujw77 Have you tested it?

**Comment by hujw77** [comment]
> @hujw77 Have you tested it?

@AurevoirXavier tesed the `Remark` call just now, and he will test `sudo` call later.

File: packages/addresses/addresses.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
         "kton": "0x1994100c58753793D52c6f457f189aa3ce9cEe94"
     },
     "43": {
+        "Root": "0x9Cd2ee66aF50398A36aAfd461AA7AdC01962E0Ae",
```
Comment (AurevoirXavier) [inline packages/addresses/addresses.json:17]: This is my pangolin test account.

Comment (AurevoirXavier) [reply packages/addresses/addresses.json:17]: We'll need another one for CRAB.

**Comment by hujw77** [comment]
@wuminzhe I draft gnosis MultiSigWallet guide in this pr.

**Pull request merged**